app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is correctly defined as a class attribute. However, ensure that it is accessed using `Person.people` instead of `self.people` within the `__init__` method to avoid confusion and potential errors.